### DISCUSSION TITLE
2023-03-12 compile error, 4.2.7 E3V2, using FIX MOUNTED PROBE

### DISCUSSION THREAD

**Discussion by Ojref** [mriscoc/Ender3V2S1#854] [General]
Hello, 
As per the title, when I start to make changes to the raw configuration for the board and probe, the second I turn off the MANUAL_PROBE and change to FIX_MOUNTED_PROBE and do a test compile I get

*** [.pio\build\STM32F103RE_creality\src\src\inc\Warnings.cpp.o] Error 1
Marlin\src\lcd\e3v2\proui\dwin.cpp: In function 'void Trammingwizard()':
Marlin\src\lcd\e3v2\proui\dwin.cpp:2535:50: error: 'BED_TRAMMING_PROBE_TOLERANCE' was not declared in this scope
 2535 |       if (ABS(MeshViewer.max - MeshViewer.min) < BED_TRAMMING_PROBE_TOLERANCE) {

Does anyone have an idea what that's from? I'm currently using MRISOC's NOV22 release of the firmware without issues.

**Comment by mriscoc** [comment]
Wrong configuration files, you need to use configurations files from the same date of the main project sources.

**Comment by Ojref** [comment]
Hello,
 To clarify, I'm using the configuration.h/configuration_adv.h that are part of the 2023-3-12 release. I'm changing from manual probe to fixed probe type in the configuration, and then building it in VSCODE to see if it compiles properly.

**Reply by mriscoc** [reply]
Use the Configurator to generate the configuration files.
https://github.com/mriscoc/Special_Configurations

**Comment by Ojref** [comment]
Thank you very much, I'm sure these type questions can be somewhat redundant but sincerely appreciated.
